IPs/RSKIP_tbd.md

```diff
@@ -0,0 +1,282 @@
+|**Title**      |Powpeg Spendability Validation Protocol |
+|**Created**    |31-AUG-23 |
+|**Author**     |JD |
+|**Purpose**    |Sec|
+|**Layer**      |Core|
+|**Complexity** |2|
+|**Status**     |Draft|
+
+## Abstract
+
+The Powpeg composition changes from time to time for different reasons, mainly members joining or leaving and/or POWHSM upgrades that require onboarding.
+The Powpeg Bitcoin address also changes when the redeemscript is upgraded, for instance when the ERP redeemscript was added.
+Everytime this happens, the Bridge undergoes a process that is irreversible, after a certain amount of blocks, the new Powpeg composition will receive all the funds and become responsible for its safety. If the newly elected Powpeg were to be unable to securely store the funds, the users could have their funds at risk.
+
+This RSKIP proposes a new process that will happen before the new Powpeg takes over, and could imply reverting the Powpeg election if the new Powpeg is considered not safe.
+
+## Motivation
+
+The Powpeg election is a critical process that should have as many reassurances as possible.
+As mentioned in the previous section, nowadays once the election takes place, the process is irreversible.
+This RSKIP proposes an automated and decentralized process to add one more layer of security to this process to avoid any human mistake putting funds at risk.
+
+This process protects funds from:
+- An invalid redeemscript that would render the funds burned.
+- A composition of pegnatories that are not responsive.
+
+This process doesn't include:
+- Verifying proposed new members use a valid HSM version.
+
+## Specification
+
+This RSKIP proposes changes in the Powpeg change process. The subsections will expand on the changes for each step.
+
+The election process in itself doesn't change. The authorized members will still propose a new composition, once the voting finishes the new composition will be committed and the Powpeg change will start.
+
+The current process consists in the following phases:
+1. Election start: m/n authorized members propose to start a new election.
+2. Composition proposal: m/n vote for the new composition.
+3. New composition commitment: m/n vote to commit new composition. New elections are disabled.
+4. Pre activation phase: immediately after commitment and during a number of blocks. No real change for end users.
+5. Activation phase: new Powpeg address activates, two-way peg starts operating from this new address.
+6. Migration phase: Funds from the retiring Powpeg wallet are moved to the active Powpeg.
+7. Post migration phase: Retiring Powpeg information is removed from the Bridge. New elections are re-enabled.
+
+### Commit new Powpeg
+
+The commitment of a new Powpeg composition remains almost unchanged except for a subtle need.
+Given the election could be reverted (see [here](#election-rollback)) the Bridge will have to store the last retired Powpeg script in a backup until the validation phase finishes.
```

Comment:
i would say _retiring_ instead of _last retired_ since in the validation phase it is not retired yet